admin_guide/allocating_node_resources.adoc

```diff
@@ -101,6 +101,50 @@ status:
 ----
 ====
 
+[[system-resources-reported-by-node]]
+== System resources reported by node
+
+Since OpenShift 3.3 each node reports system resources utilized by the container runtime and kubelet.
+To better aid your ability to configure *--system-reserved* and *--kube-reserved*,
+you can introspect the nodes resource usage using the node summary api which is accessible at *<master>/api/v1/nodes/<node>/proxy/stats/summary*.
```

Comment:
Right, "you can introspect corresponding node's resource usage" it is as it refers to configuration of a single node.